rclpy/rclpy/__init__.py

```diff
@@ -165,6 +166,9 @@ def create_node(
         This option doesn't affect `parameter_overrides`.
     :param automatically_declare_parameters_from_overrides: If True, the "parameter overrides" will
         be used to implicitly declare parameters on the node during creation, default False.
+    :param enable_logger_service: ``True`` if ROS2 services are created to allow external nodes
+        to get and set logger levels of this node. Otherwise, loggers are only configured locally.
+        That is, logger levels cannot be changed remotely.
```

Comment:
```suggestion
        to get and set logger levels of this node. Otherwise, logger levels are only managed locally.
        That is, logger levels cannot be changed remotely.
```